doc/source/structures/volumes_and_files/volume.rst

```diff
@@ -34,27 +34,31 @@ Represents a :struct:`kOSProcessor` hard disk or the archive.
 
         * - :attr:`FILES`
           - :struct:`Lexicon`
-          - Lexicon of all files on the volume
+          - Lexicon of all files and directories on the volume
 
         * - :attr:`POWERREQUIREMENT`
           - :ref:`scalar <scalar>`
           - Amount of power consumed when this volume is set as the current volume
```

Comment:
I tried to change as little as possible when it comes to electric charge consumption. I should have some time tomorrow to look into it and give you a better answer.